javascript/javascript1/week1/index.js

```diff
@@ -3,29 +3,26 @@
 const yearOfBirth = 1983;
 const yearFuture = 2023;
 const age = yearFuture - yearOfBirth;
-const result = (`You will be ${age} years old in ${yearFuture}.`);
+const result = `You will be ${age} years old in ${yearFuture}.`;
 /* or const result = ("You will be " + age + " years old in" + yearFuture + ".") */
 
 console.log(result);
 
```

Comment:
Super nice and clean code. You could have added the variable data types in a comment as that was asked in the task :)

Reply:
I haven't payed attention to those questions, to be frank. 
1. Task - Age-ify (A future age calculator)
    yearOfBirth and yearFuture are both number types.
    
2. Task - Goodboy-Oldboy (A dog age calculator) 
   shouldShowResultInDogYears is a boolean type. 
   
There we go ! 

 Thank you for your time to review my code.